packages/compat/src/standalone-addon-build.ts

```diff
@@ -52,7 +52,12 @@ ${summarizePeerDepViolations(violations)}
   });
 
   let fakeTargets = Object.values(index.packages).map(dir => {
-    return writeFile(join(dir, '..', 'moved-package-target.js'), '');
+    let segments = dir.split('/');
```

Comment:
@ef4 is it ok for this to not split on `sep` ?

Reply:
I actually tried that first and it breaks, because the index of moved packages here is always assembled in unix format.